src/main/java/frc/robot/subsystems/CameraSubsystem.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.cameraserver.CameraServer;
+import edu.wpi.first.cscore.CvSink;
+import edu.wpi.first.cscore.UsbCamera;
+import edu.wpi.first.cscore.CvSource;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import org.opencv.core.Mat;
+import org.opencv.core.Point;
+import org.opencv.core.Scalar;
+import org.opencv.imgproc.Imgproc;
+
+public class CameraSubsystem extends SubsystemBase {
+  Thread m_visionThread;
+  /** Creates a new VideoOverlaySubsystem. */
```

Comment:
Since this is hard to read code, I'm going to require a method description about what this does (or intends to do) at a high level.

Reply:
Added a method description explaining the function of the thread.